### PR TITLE
V1.2

### PR DESCRIPTION
**Pull request by xfynx**
+ feature flags
+ token expiration and refresh methods
+ improvements on claims methods
+ specification update
+ README and spec updates

**Note:** feel free to skip review on everything that lies under directory `./kinde_api` - it is auto-generated.

### CODE REVIEW

**Comment by xfynx** [comment]
hey @abd827!
this code is generated by openapi script (all that lies under directory `./kinde_api`), so we can't handle that easily. it requires us to rewrite part of the code or something like. i already use some replacements in our scripts, but it is only for api_client and connection which are not going to change over specification changes. But everything else tied with specs.

i'm too not a big fan of codestyle produced by openapi, but it is what it is 🤷 at least it works well

**Review by rairaman — APPROVED** [review]
LGTM

File: kinde_api/lib/kinde_api/api/environments_api.rb [hunk 1]
```diff
@@ -139,6 +139,63 @@ def delete_environement_feature_flag_overrides_with_http_info(opts = {})
...
+    # List environment feature flags
+    # Get environment feature flags.
+    # @param [Hash] opts the optional parameters
+    # @return [Array<(GetEnvironmentFeatureFlagsResponse, Integer, Hash)>] GetEnvironmentFeatureFlagsResponse data, response status code and response headers
+    def get_environement_feature_flags_with_http_info(opts = {})
+      if @api_client.config.debugging
+        @api_client.config.logger.debug 'Calling API: EnvironmentsApi.get_environement_feature_flags ...'
+      end
+      # resource path
+      local_var_path = '/api/v1/environment/feature_flags'
```
Comment (abd827) [inline kinde_api/lib/kinde_api/api/environments_api.rb:160]: if the resource path will stay the same then wouldn't it be better to have a constant initialised for it

**Review by abd827 — APPROVED** [review]

File: kinde_api/lib/kinde_api/api/environments_api.rb (hunk 1 above)
Comment (xfynx) [reply kinde_api/lib/kinde_api/api/environments_api.rb:160]: well, again - it is generated code. and, unlike files like `api_client.rb` and `connection.rb`, these kind of files can be changed in any time with specification change, so we can't replace and modify it easily.

code is not perfect, i understand, but it comes out of openapi generator

**Review by abd827 — APPROVED** [review]

**Pull request merged**